.github/workflows/ci.yml

```diff
@@ -4,144 +4,88 @@ on:
+      - run: npm run lint
 
-
-  test-addon-floating:
-    name: Test addon (floating dependencies)
+  floating-dependencies:
+    name: Floating Dependencies
     runs-on: ubuntu-latest
-    timeout-minutes: 5
-    steps:
-      - name: Check out a copy of the repo
-        uses: actions/checkout@v2
 
-      - name: Use Node.js ${{ env.NODE_VERSION }}
-        uses: actions/setup-node@v2-beta
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
         with:
           node-version: ${{ env.NODE_VERSION }}
 
-      - name: Install dependencies
-        run: npm install --no-package-lock
+      - run: npm i -g npm@7
+      - run: npm install --no-package-lock
+      - run: npm test
 
-      - name: Test
-        run: npm run test:ember
+  test:
+    name: Tests
+    runs-on: ubuntu-latest
+    env:
+      PERCY_PARALLEL_NONCE: ${{ github.run_id }}-${{ github.run_number }}
+      PERCY_PARALLEL_TOTAL: 1
+      PERCY_TOKEN: e01f89bea06d5ccd1207a5cd2799645e4942ecd0d8da739502a206d0fc7392fc
+      RUN_PERCY_TESTS: true
 
 
-  test-addon-locked:
-    name: Test addon (locked dependencies)
-    runs-on: ubuntu-latest
-    timeout-minutes: 5
     steps:
-      - name: Check out a copy of the repo
-        uses: actions/checkout@v2
-
-      - name: Use Node.js ${{ env.NODE_VERSION }}
-        uses: actions/setup-node@v2-beta
+      - uses: actions/checkout@v2
+      - uses: mansona/npm-lockfile-version@v1
+      - uses: actions/setup-node@v2
         with:
           node-version: ${{ env.NODE_VERSION }}
 
-      - name: Cache npm cache and node_modules
-        id: cache-dependencies
-        uses: actions/cache@v2
+      - run: npm i -g npm@7
+      - run: npm ci
+      - uses: percy/exec-action@v0.3.1
         with:
-          path: |
-            ~/.npm
-            node_modules
-          key: ${{ runner.os }}-${{ env.NODE_VERSION }}-${{ hashFiles('**/package-lock.json') }}
-          restore-keys: ${{ runner.os }}-${{ env.NODE_VERSION }}-
-
-      - name: Install dependencies
-        run: npm install
-        if: steps.cache-dependencies.outputs.cache-hit != 'true'
-
-      - name: Test
-        uses: percy/exec-action@v0.3.1
-        with:
-          custom-command: npm run test:ember
-        env:
-          PERCY_PARALLEL_NONCE: ${{ env.PERCY_PARALLEL_NONCE }}
-          PERCY_PARALLEL_TOTAL: ${{ env.PERCY_PARALLEL_TOTAL }}
-          PERCY_TOKEN: e01f89bea06d5ccd1207a5cd2799645e4942ecd0d8da739502a206d0fc7392fc
-          RUN_PERCY_TESTS: true
-
+          custom-command: npm run test
 
-  test-compatibility:
-    name: Test compatibility
+  try-scenarios:
+    name: "ember-try: ${{ matrix.ember-try-scenario }}"
     runs-on: ubuntu-latest
+    needs: test
+
     strategy:
-      fail-fast: true
       matrix:
-        scenario:
+        ember-try-scenario:
           - 'ember-lts-3.4'
           - 'ember-lts-3.8'
-          - 'ember-lts-3.12'
-          - 'ember-lts-3.16'
-          - 'ember-lts-3.20'
```

Comment:
I didn't see why these scenarios were removed but 3.4 and 3.8 kept. I think 3.20 should be included since it's an active LTS.

Reply:
This config keeps the versions lesser than or equal to the addon’s version. A follow up PR could update the Ember version and this config, however we are trying to put most of the effort towards the redesign, so I’m not sure it’s worth doing that right now.